lib/routes/initial_walkthrough/initial_walkthrough.dart

```diff
@@ -161,10 +162,12 @@ class InitialWalkthroughPageState extends State<InitialWalkthroughPage>
 
     final themeProvider = ThemeProvider.controllerOf(context);
     try {
-      String mnemonic = bip39.generateMnemonic(strength: 128);
-      await accountBloc.startNewNode(mnemonic: mnemonic);
+      await accountBloc.connect(mnemonic: mnemonic ?? bip39.generateMnemonic(strength: 128));
     } catch (error) {
-      _log.i("Failed to register node", ex: error);
+      _log.i("Failed to ${isRestore ? "restore" : "registe"} node", ex: error);
+      if (isRestore) {
```

Comment:
We don't need this fallback here to restore, do we?
The connects should already take that into account, if it is passed with mnemonics it restores, if it is passed with empty mnemonics it will look into the secure storage for mnemonics.

Reply:
We do, this flow uses the mnemonics that the user has entered manually.
If it fails, EnterMnemonicsPage is opened with the same mnemonics.

How connect on AccountBloc works is:
If mnemonics is supplied, store the mnemonics. This parameter is used to differentiate if we're establishing connection with an existing node or not.
Then later on when calling the SDK's connect API, mnemonics are restored from secure storage.

Reply:
I see now. Thanks.